### ISSUE TITLE
Use displayName instead of username in AddonsByAuthorsCard

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#4742]
The `AddonsByAuthorsCard` component accepts an array of author names to fetch an add-on by. But it should get the author's displayName from state in order to display that in the `More add-ons by ${authorName}` text. Right now it shows their username which looks bad, and inconsistent because we show the display name above, underneath the add-on name on the details page:

![name](https://user-images.githubusercontent.com/1583842/38318929-3f09fa20-3839-11e8-9a18-c37a9e766b1e.png)

**Comment by tofumatt** [comment]
I don't think this would be fixed yet, but I think that username is the same as the `displayName` or displayName is missing.

**Comment by daiict218** [comment]
The user is same as you posted in the description of the issue. Also, I debugged locally and I was getting a field of `name` and `username` in author of addon and it has been resolved through `name` which can be `username`. In case of above screenshot both `name` and `username` were same

**Comment by wagnerand** [comment]
@bobsilverberg and I are pretty sure this is a dupe of #4806 which is a dupe of #4796 (oh my).

@ioanarusiczki would you mind testing this (on dev or stage) to double check?

**Comment by ioanarusiczki** [comment]
The remaining issue is https://github.com/mozilla/addons-server/issues/6964
